css/styles.css

```diff
@@ -0,0 +1,33 @@
+body {
+  background-color: #df4747;
+  color: aliceblue;
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: "Overpass", sans-serif;
+  font-size: 17px;
+  list-style: none;
+}
+
+h1 {
+  font-family: "Dancing Script", cursive;
+  font-size: 60px;
+}
+
+h3 {
+  font-family: "Dancing Script", cursive;
+  font-size: 25px;
+}
```

Comment:
Try to use `rem` over `px` for `font-size`s: https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css